client/app/scripts/reducers/root.js

```diff
@@ -421,7 +421,12 @@ export function rootReducer(state = initialState, action) {
       // highlight edge
       state = state.update('highlightedEdgeIds', (highlightedEdgeIds) => {
         highlightedEdgeIds = highlightedEdgeIds.clear();
-        return highlightedEdgeIds.add(action.edgeId);
+        highlightedEdgeIds = highlightedEdgeIds.add(action.edgeId);
+        if (action.bidirectional) {
+          const opposite = action.edgeId.split(EDGE_ID_SEPARATOR).reverse().join(EDGE_ID_SEPARATOR);
+          highlightedEdgeIds = highlightedEdgeIds.add(opposite);
+        }
+        return highlightedEdgeIds;
```

Comment:
Suggesting that because keeping track of bidirectionality seems kinda unnecessary if it doesn't reflect the way edges are rendered.